src/Data/Time/PreciseDuration/Internal.purs

```diff
@@ -31,37 +31,37 @@ instance showPreciseDuration :: Show PreciseDuration where
   show (Days d) = "(Days " <> show d <> ")"
   show (Weeks d) = "(Weeks " <> show d <> ")"
 
+unwrapPreciseDuration :: PreciseDuration -> Decimal
```

Comment:
I think this should be called something like `toDecimalLossy`.

Reply:
I don't think `toDecimalLossy` conveys the right intent for this -- what are you losing? what does the decimal you get mean? `unwrap` makes it clear that you're just removing the constructor.

Reply:
You aren't just removing the constructor. You are removing all constructors and losing any context for what the `Decimal` means.

Reply:
ie, using this function with a Milliseconds constructor vs a Days constructor both yields a Decimal, but they mean very different things. Therefore it's lossy.

Reply:
ok